lib/system/gc.nim

```diff
@@ -263,7 +263,7 @@ proc nimGCunrefNoCycle(p: pointer) {.compilerProc, inline.} =
 proc nimGCunrefRC1(p: pointer) {.compilerProc, inline.} =
   decRef(usrToCell(p))
 
-proc asgnRef(dest: PPointer, src: pointer) {.compilerProc, inline.} =
+proc asgnRef(dest: PPointer, src: pointer) {.compilerRtl, inl.} =
```

Comment:
This is quite expensive. The code was written in order to support ``inline`` in DLL builds. Only procs that access ``gch`` directly need to go through the DLL indirection.